examples/simple_example/main.tf

```diff
@@ -14,13 +14,42 @@
  * limitations under the License.
  */
 
-provider "google" {
-  version = "~> 2.0"
+# Required if using User ADCs (Application Default Credentials) for Cloud Identity API.
+# provider "google-beta" {
+#   version = "~> 3.0"
+#   user_project_override = true
+#   billing_project       = var.project_id
+# }
+
+resource "google_service_account" "manager" {
+  project      = var.project_id
+  account_id   = "example-manager"
+  display_name = "example-manager"
+}
+
+resource "google_service_account" "member" {
+  project      = var.project_id
+  account_id   = "example-member"
+  display_name = "example-member"
+}
+
+module "child_group" {
+  source = "../.."
+
+  id           = "example-child-group@${var.domain}"
+  display_name = "example-child-group"
+  description  = "Example child group"
+  domain       = var.domain
+  managers     = ["${google_service_account.manager.account_id}@${var.project_id}.iam.gserviceaccount.com"]
```

Comment:
@xingao267 I think you can use google_service_account.manager.email.

Reply:
The underlying `for_each` would complain it's a computed value.

Reply:
I'm surprised that account_id isn't computed, but got it.

Reply:
Yeah I was checking this too for https://github.com/terraform-google-modules/terraform-google-group/pull/1#discussion_r527971597 but as we pass in the `account_id` to the resource it is not computed. `unique_id` is the computed value.